include/rocksdb/env_encryption.h

```diff
@@ -194,13 +196,235 @@ class CTREncryptionProvider : public EncryptionProvider {
                                           size_t blockSize);
 
   // CreateCipherStreamFromPrefix creates a block access cipher stream for a
-  // file given given name and options. The given prefix is already decrypted.
+  // file given
+  // given name and options. The given prefix is already decrypted.
   virtual Status CreateCipherStreamFromPrefix(
       const std::string& fname, const EnvOptions& options,
       uint64_t initialCounter, const Slice& iv, const Slice& prefix,
       std::unique_ptr<BlockAccessCipherStream>* result);
 };
 
+class EncryptedSequentialFile : public SequentialFile {
+ private:
```

Comment:
I personally prefer protected in nearly every scenario because it simplifies future extension and unit test wrappers.  I believe "private" was from original authors.